### PR TITLE
Feature flags API

### PR DESCRIPTION
**Pull request by mgritter**

### CODE REVIEW

File: website/content/api-docs/system/internal-ui-feature.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+layout: api
+page_title: /sys/internal/ui/feature-flags - HTTP API
+sidebar_title: <code>/sys/internal/ui/feature-flags</code>
+description: >-
+  The `/sys/internal/ui/feature-flags` endpoint exposes feature flags to the UI.
+---
+
+# `/sys/internal/ui/feature-flags`
+
+The `/sys/internal/ui/feature-flags` endpoint is used to exppose feature flags
```
Comment (swayne275) [inline website/content/api-docs/system/internal-ui-feature.mdx:11]:
```suggestion
The `/sys/internal/ui/feature-flags` endpoint is used to expose feature flags
```

**Review by swayne275 — COMMENTED** [review]
typo but other than that i'll approve

**Review by swayne275 — APPROVED** [review]
Looks good!

**Comment by mgritter** [comment]
Hm, looks like website build failed, will investigate

**Pull request merged**